### DISCUSSION TITLE
Web Interface Issues

### DISCUSSION THREAD

**Discussion by RJCato** [Security-Onion-Solutions/securityonion#11489] [2.4]
### Version

2.4.20

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

12

### RAM

64

### Storage for /

200GB

### Storage for /nsm

200GB

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I have fully installed Security Onion. I am trying to log into the web interface but it keeps timing out. I followed all steps in the YouTube video for the setup procedure. The only thing I did differently was use DHCP instead of static IP. This was because static would not work after multiple tries. DHCP did work however. Whenever I log in, sobridge port (number) (device_name) enters blocking state, disabled state, blocking state, forwarding state, etc. I do not know if this is normal. I ran sudo so-status and everything is running. Everything is up to date and the IPs are correct. I do not know if this would affect anything, but my vm manager is on my ssd while the SO software is installed on my hhd. I have been watching the tutorial videos on YouTube to set up the security onion software , which is how I have gotten this far. I left network 1 set to NAT and set network 2 to internal network and promiscuous mode is set to allow all. Everything seems to be working correctly from what I can tell, but when I enter the IP to pull up the web interface, it times out. I apologize if I am just overlooking something super simple as I am completely new to this.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
> I am trying to log into the web interface but it keeps timing out. I followed all steps in the YouTube video for the setup procedure. 

Did you allow your web browser to connect through the host-based firewall? From https://docs.securityonion.net/en/2.4/firewall.html#configuration:
_If for some reason you can’t access [Security Onion Console (SOC)](https://docs.securityonion.net/en/2.4/soc.html#soc), you can use the so-firewall command to allow your IP address to connect (replacing IPADDRESS with your actual IP address):
so-firewall includehost analyst IPADDRESS_

> The only thing I did differently was use DHCP instead of static IP. This was because static would not work after multiple tries. DHCP did work however. 

From https://docs.securityonion.net/en/2.4/hardware.html#nic:
_We recommend using static IP addresses where possible._

> Whenever I log in, sobridge port (number) (device_name) enters blocking state, disabled state, blocking state, forwarding state, etc. I do not know if this is normal. 

Yes, this is normal:
https://docs.securityonion.net/en/2.4/console.html

> I ran sudo so-status and everything is running. Everything is up to date and the IPs are correct. I do not know if this would affect anything, but my vm manager is on my ssd while the SO software is installed on my hhd. 

We recommend ssd for best performance but even with hdd you should at least be able to connect to the web interface.

> I have been watching the tutorial videos on YouTube to set up the security onion software , which is how I have gotten this far. I left network 1 set to NAT and set network 2 to internal network and promiscuous mode is set to allow all. Everything seems to be working correctly from what I can tell, but when I enter the IP to pull up the web interface, it times out. I apologize if I am just overlooking something super simple as I am completely new to this.

Another thing to check is to see if perhaps your interfaces are swapped.

**Reply by RJCato** [reply]
Ok. So the reason I am having to use DHCP is because if I use static, the install encounters an error. DHCP was the only time I had success. Should I use airgap? I am selecting the right IPs and everything. It feels like something that is kind of hard to make a mistake on, but it just will not work. I actually experimented (I have been watching a ton of videos to try different methods to get it to work). I ended up attempting to install the standalone changing Network 1 to bridged adapter and Network 2 to host only adapter. It had better success installing this time. Most of the time it would count with the timer to 20-ish seconds, try several more times, and give up with the eval.

**Reply by dougburks** [reply]
> So the reason I am having to use DHCP is because if I use static, the install encounters an error. DHCP was the only time I had success.

If DHCP works but static IP address results in an error, then that would suggest that there is something wrong with the static IP information and you are unable to access the Internet.

> Should I use airgap?

You could use airgap but you would have to wait for us to release a new ISO image to get any updates. The ideal solution would be to figure out the static IP issue so that you have full Internet access and can get daily updates without having to wait.
